### PR TITLE
[Merged by Bors] - chore: remove redundant LinearEquiv.map_neg/sub

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
These are redundant with `_root_.{map_neg,map_sub}`.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by grunweg — APPROVED** [review]
Looks good to me.

File: Mathlib/LinearAlgebra/Basic.lean [hunk 1]
```diff
@@ -897,15 +897,8 @@ variable {re₁₂ : RingHomInvPair σ₁₂ σ₂₁} {re₂₁ : RingHomInvPai
...
-theorem map_neg (a : M) : e (-a) = -e a :=
-  e.toLinearMap.map_neg a
-#align linear_equiv.map_neg LinearEquiv.map_neg
-
--- @[simp] -- Porting note (#10618): simp can prove this
-theorem map_sub (a b : M) : e (a - b) = e a - e b :=
-  e.toLinearMap.map_sub a b
-#align linear_equiv.map_sub LinearEquiv.map_sub
+#align linear_equiv.map_neg map_negₓ
+#align linear_equiv.map_sub map_subₓ
```
Comment (loefflerd) [inline Mathlib/LinearAlgebra/Basic.lean:901]: What's the significance of the little `x` here?

Comment (grunweg) [reply Mathlib/LinearAlgebra/Basic.lean:901]: The porting wiki contains this:
"when mathport suggests that you add ₓ in the name of an #align statement, figure out why it wants that and leave a porting note. Common reasons are:
- a change in the order of universe variables, typeclass arguments or implicit arguments
- some other defeq change, due to a change deep down in core or std."

Comment (Ruben-VandeVelde) [reply Mathlib/LinearAlgebra/Basic.lean:901]: It means that the types don't match up exactly; in this case the new lemma is much more general.

**Comment by erdOne** [comment]
I think this is useful to keep as sometimes `_root_.map_sub` times out when synthesizing typeclass arguments.

**Comment by Ruben-VandeVelde** [comment]
I have not experienced that

**Comment by riccardobrasca** [comment]
Thanks!

bors merge

**Pull request closed without merging**